src/main/frontend/state.cljs

```diff
@@ -1432,30 +1326,6 @@
   [_repo files? id]
```

Comment:
This fn sets state in frontend.db/persist! that is no longer used as evidenced by the removal below. Should we remove this unused state and fn?

Reply:
Yes, we should remove both.